### PR TITLE
chore: updated to RN 0.71

### PR DESCRIPTION
**Pull request by pinpong**
updated to react-native 0.71

### CODE REVIEW

File: example/ios/Podfile [hunk 1]
```diff
@@ -2,9 +2,24 @@ require_relative '../node_modules/react-native/scripts/react_native_pods'
...
+#   dependencies: {
+#     ...(process.env.NO_FLIPPER ? { 'react-native-flipper': { platforms: { ios: null } } } : {}),
+# ```
+flipper_config = ENV['NO_FLIPPER'] == "1" ? FlipperConfiguration.disabled : FlipperConfiguration.enabled
+
+linkage = ENV['USE_FRAMEWORKS']
+if linkage != nil
+  Pod::UI.puts "Configuring Pod with #{linkage}ally linked Frameworks".green
+  use_frameworks! :linkage => linkage.to_sym
+end
```
Comment (Simon-TechForm) [inline example/ios/Podfile:22]: What is all this? Not a part of the official react-native template.

File: android/build.gradle [hunk 2]
```diff
@@ -12,42 +12,33 @@ buildscript {
...
 
 dependencies {
   def work_version = safeExtGet('targetSdkVersion', 27) < 31 ? "2.5.0" : "2.7.1"
   implementation 'com.facebook.react:react-native:+'
   implementation "com.google.android.gms:play-services-base:${safeExtGet('playServicesVersion', '18.1.0')}"
-  implementation "com.google.android.gms:play-services-maps:${safeExtGet('playServicesVersion', '18.0.2')}"
+  implementation "com.google.android.gms:play-services-maps:${safeExtGet('playServicesVersion', '18.1.0')}"
   implementation "com.google.android.gms:play-services-location:20.0.0"
-  implementation 'com.google.maps.android:android-maps-utils:0.5'
+  implementation 'com.google.maps.android:android-maps-utils:3.3.0'
```
[the remark this answers is not in the document]
Comment (pinpong) [reply android/build.gradle:42]: Running this version in production for long time without problem. Can you explain what are the breaking changes?

[the remark this answers is not in the document]
Comment (Simon-TechForm) [reply android/build.gradle]: You can read about the changes [here](https://github.com/googlemaps/android-maps-utils#migration-guide-from-v0x-to-10) :)

**Comment by Simon-TechForm** [comment]
Are you okay with me closing this for now? Once [detox](https://github.com/wix/Detox/issues/3884) supports 0.71 I'll be happy to review a pr that bumps to 0.71 :)

**Pull request closed without merging**